### PR TITLE
Update react to v17

### PR DESCRIPTION
**Pull request by roosnic1**

### CODE REVIEW

File: packages/editor/src/client/atoms/typographicTextArea.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import React, {
```
Comment (bishoymelek) [inline packages/editor/src/client/atoms/typographicTextArea.tsx:1]: There is an issue in this file now: 
Screenshot from 2021-07-20 12-06-41
You can reproduce it in article edit page => add teasers, choose an article and press on save.

**Review by bishoymelek — APPROVED** [review]

File: packages/editor/src/client/atoms/typographicTextArea.tsx (hunk 1 above)
Comment (roosnic1) [reply packages/editor/src/client/atoms/typographicTextArea.tsx:1]: @bishoymelek i can't reproduce the error. Maybe i'm doing it wrong. What did you mean with "in article edit page"???

File: packages/editor/src/client/atoms/helpers.ts [hunk 2]
```diff
@@ -1,40 +1,4 @@
-import {CSSStyle} from '@karma.run/react'
-
-import {
```
[the remark this answers is not in the document]
Comment (roosnic1) [reply packages/editor/src/client/atoms/helpers.ts:3]: @bishoymelek was not needed anymore. The original author implemented something here that he never reused anywhere else. And it doesn't work with React 17 anymore.

File: packages/editor/src/client/atoms/typographicTextArea.tsx (hunk 1 above)
Comment (roosnic1) [reply packages/editor/src/client/atoms/typographicTextArea.tsx:1]: @bishoymelek fixed it

**Review by bishoymelek — APPROVED** [review]
:tada:

**Pull request merged**